src/multipleObserver.js

```diff
@@ -1,5 +1,12 @@
 import React from "react"
 
+// eslint-disable-next-line no-unused-vars
+import { ComponentWrapper, StoreConfig } from "./types"
```

Comment:
Shall we import JSDoc definitions at all? 🤔

Reply:
There is got to be a better way, I agree.

Going to try to convert types into `.d.ts` files, seems like this approach is more popular when it comes to providing package types.